Jumpscale/clients/blockchain/tfchain/TFChainTime.py

```diff
@@ -0,0 +1,12 @@
+# BotMonth is defined as 30 days of exactly 24 hours, expressed in seconds.
+BotMonth = 60 * 60 * 24 * 30
+CompactTimestampAccuracyInSeconds = 60
+MaxBotPrepaidMonths = 24
+
+
+class TFChainTime:
```

Comment:
I had it in a separate class for several reasons:
- I wanted it to be consistent with jumpscale_lib. 
- In jumpscale_lib, the time module has more than one function actually. And I need it to be available without having to get a wallet instance because we need it in the reservation template and we dont have access to the wallet instance and we don't need to.
- I feel like having the module variables and the class in a separate makes things more clear and makes it more obvious that these are calculations related to tfchain specifically. And any other module or template can use them without having to create a wallet or so. I already feel like some of the tfchain files are too big and too confusing with too many classes in the same file.

Reply:
Ok I see. So we can leave this class then.